js/ui/camera.js

```diff
@@ -296,7 +296,8 @@ class Camera extends Evented {
      * Pans and zooms the map to contain its visible area within the specified geographical bounds.
      *
      * @memberof Map#
-     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into.
+     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into. If bounds contains
+     *     only one point, the map center transitions to that point at zoom level options.maxZoom or {@link Map#getMaxZoom}.
```

Comment:
It used to error on `LngLatBounds` where ne + sw were the same point, this PR fixes that because that case is technically a valid instance of [`LngLatBounds`](https://www.mapbox.com/mapbox-gl-js/api/#LngLatBounds). I could make the docs language more clear to that effect. I don't think we should accept `LngLatLike` because I don't think its a recommended use-case for `fitBounds` 😕